packages/legend-art/src/components/panel/PanelListSelectorItem.tsx

```diff
@@ -50,6 +52,7 @@ export const PanelListSelectorItem: React.FC<{
       >
         {title}
       </div>
+      {children}
```

Comment:
This looks strange, `children` is the ultimate escape hatch when people want to fully customize the list item, as such, if `chilren` is provided, we won't need `title` and vice-versa. What choice do you make here?